_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -57,12 +57,18 @@ _TBA: to be announced_
 
 The description above only describes Bitcoin Core software releases. Many other parts of the Bitcoin system contain their own versions.  A few examples:
 
-- Every **block** and **transaction** contains a version number.
+- Every **transaction** contains a version number.
+- Every **block** allows miners to signal their readiness to enforce new
+  versions of the consensus rules.
 - The **P2P network protocol** uses version numbers to allow nodes to announce what features they support.
 - Bitcoin Core's **built-in wallet** has its own internal version number.
 
 These versions numbers are deliberately decoupled from Bitcoin Core's version number as the Bitcoin Core project either has no direct control over them (as is the case with blocks and transactions), or tries to maintain compatibility with other projects (as is the case with the network protocol), or allows for the possibility that no major changes will be made in some releases (as is sometimes the case with the built-in wallet).
 
+The consensus protocol itself doesn't have a version number, although it
```

Comment:
I'm really not sure about this para on the basis it opens up another whole question of "zomg why isnt the consensus layer versioned".